### ISSUE TITLE
Use Windows notifications

### ISSUE THREAD

**Issue by kaiser-chris** [gyunaev/birdtray#226]
It would be nice to enable win api notifications in the windows build. This will for example allow users to use the notification center on windows 10 for missed notifications.

I understand the custom notifications are needed for platform independece and some users prefer them, but I think it would be nice to have a config option to use win api notifications.

References:

- https://docs.microsoft.com/en-us/windows/win32/shell/notification-area (Old Windows compatible)
- https://github.com/mohabouje/WinToast (lib for Windows 8/10 notifications)

**Comment by Abestanis** [comment]
Birdtray doesn't send any notifications, as far as I know. If you are referring to the notifications about new Emails, those are sent by Thunderbird and unfortunately we have no control over them.

There is a [feature request for Thunderbird](https://bugzilla.mozilla.org/show_bug.cgi?id=1192615) that requests native notifications.

**Comment by gyunaev** [comment]
Technically we can extract the new email info from Mork, it's there.

**Comment by Abestanis** [comment]
Yes, if we want to send our own notifications. Should be possible.

**Comment by rnc** [comment]
I think this is similar (from the Linux POV) to #122 

As the thunderbird native notifications appeared to be completed unreliable for me (Fedora 31) I switched to https://addons.thunderbird.net/en-US/thunderbird/addon/mailbox-alert / https://github.com/tjeb/Mailbox-Alert which allows me to configure notify-send (from libnotify) to do much better notifications.

There appears to be similar cmd line apps available for Windows to use the same pattern ( e.g. https://github.com/go-toast/toast , https://github.com/KDE/snoretoast )
